pyproject.toml

```diff
@@ -21,6 +21,7 @@ maintainers = [
     { name = "Ryan Abernathey" }
 ]
 requires-python = ">=3.10"
+# If you add a new dependency here, please also add it to .pre-commit-config.yml
```

Comment:
Thanks for adding this comment. This is a bit of a bummer but it seems there isn't a better way to keep these in sync.